### DISCUSSION TITLE
Forgot to create brew file before wiping HDD and installing Big Sur

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#2250] [Everyday usage]
Is there a way to use the `brew bundle dump`  command to get the installed packages on another drive? My user folder on my backup drive.

(I moved from Catalina to Big Sur and forgot to create a brew file beforehand. Now I can't startup from my computer from the backup drives containing Catalina OS, or my Time Machine drive while running Big Sur. Plan was to start up from the backup and create the brew file but no go after two days of trying.)

**Comment by SMillerDev** [comment]
You can't run `brew bundle` on an OS that isn't installed, but if you included `/usr/local` in your backups for some reason you could look in `/usr/local/opt` for that installed package names and write a `Brewfile` manually that way.
